benefits/core/models/enrollment.py

```diff
@@ -10,6 +10,8 @@
 from .common import PemData, SecretNameField, template_path
 from .claims import ClaimsProvider
 from .transit import TransitAgency
+from benefits.core.context import SystemName
+from benefits.in_person.context import eligibility
```

Comment:
I know we're wanting to discuss the naming for this copy context stuff. For now, I am just following the existing naming pattern but am aware that we don't necessarily like it.

Reply:
We discussed this today in dev workshop and came up with a plan! See #2697